tests/data/hostgaplugin/vm_settings.json

```diff
@@ -1,21 +1,21 @@
 {
-    "hostGAPluginVersion":"1.0.8.115",
-    "vmSettingsSchemaVersion":"0.0",
-    "activityId": "2e7f8b5d-f637-4721-b757-cb190d49b4e9",
-    "correlationId": "1bef4c48-044e-4225-8f42-1d1eac1eb158",
-    "extensionsLastModifiedTickCount": 637693267431616449,
+    "hostGAPluginVersion": "1.0.8.115",
+    "vmSettingsSchemaVersion": "0.0",
+    "activityId": "a33f6f53-43d6-4625-b322-1a39651a00c9",
+    "correlationId": "9a47a2a2-e740-4bfc-b11b-4f2f7cfe7d2e",
+    "extensionsLastModifiedTickCount": 637726657706205217,
     "extensionGoalStatesSource": "Fabric",
-    "StatusUploadBlob": {
+    "statusUploadBlob": {
```

Comment:
I know we handle this on our end, but is the casing not guaranteed from HostGA? I remember Rohit mentioning something about fixing this, is that not in yet?

Reply:
we need to keep doing this until all the nodes in the fleet are updated with the newest HostGAPlugin (which will take a loong time, unfortunately)

Reply:
totally understand that, was just curious to know if this was being handled by HostGA or not. Like if its supposed to be handled properly by HostGA and still we're seeing discrepancy we should probably let Rohit know about it too as it might be a regression there

Reply:
At some point we should be ready to declare "version W.X.Y.Z of the HostGAPlugin is ready and all the fleet is updated"... at that point we'll add a single check for that version and remove all those workarounds/checks (case-insensitive dictionary, check for > 112, comparison of goal states, etc). Discrepancies would pop up then. Right now it is not worth checking the case of properties).